layers/+intl/japanese/README.org

```diff
@@ -0,0 +1,79 @@
+#+TITLE: Japanese layer
```

Comment:
Please checkout [conventions on spacing in documentation](https://github.com/syl20bnr/spacemacs/blob/develop/doc/CONVENTIONS.org#spacing-in-documentation) and fix any spacing problems.